js/src/redux/reducers/activeKeyword.js

```diff
@@ -0,0 +1,22 @@
+import { SET_ACTIVE_KEYWORD } from "../actions/activeKeyword";
+
+const INITIAL_STATE = null;
+
+/**
+ * A reducer for the active keyword.
+ *
+ * @param {Object} state The current state of the object.
+ * @param {Object} action The current action received.
+ *
+ * @returns {Object} The state.
+ */
+function activeKeywordReducer( state = INITIAL_STATE, action ) {
+	switch( action.type ) {
```

Comment:
Minor: not sure using a switch for just one case + default is the best option, maybe it is used just for consistency?

Reply:
Yes, It's because all actions flow through all reducers.